google-cloud-spanner/src/test/java/com/google/cloud/spanner/ChannelUsageTest.java

```diff
@@ -231,6 +235,9 @@ public void testCreatesNumChannels() {
   @Test
   public void testUsesAllChannels() throws InterruptedException, ExecutionException {
     try (Spanner spanner = createSpannerOptions().getService()) {
+      assumeFalse(
+          "GRPC-GCP is currently not supported with multiplexed sessions",
+          isMultiplexedSessionsEnabled(spanner));
```

Comment:
```suggestion
          enableGcpPool && isMultiplexedSessionsEnabled(spanner));
```

Reply:
I had originally done this. But it still introduced a bit of flaky behavior even without gcp-pool. I will be following up to re-enable this test pending some investigation.